packages/fyllut-backend/src/routers/api/helpers/sendInn.ts

```diff
@@ -145,6 +147,7 @@ export const assembleSendInnSoknadBody = (
       label: translate(attachment.label),
       beskrivelse: translate(attachment.beskrivelse),
       tittel: translate(attachment.tittel),
+      ...(attachment.vedleggskjema && { vedleggsurl: `${config.fyllutFrontendUrl}/${attachment.vedleggskjema}` }),
```

Comment:
validering av vedleggskjema? format for input?

Reply:
Rune bekreftet at formatet på input skulle være samme som det er i urlen og Lars Olav jobber med ny vedleggskomponent der det legges til validering på feltet